Umbraco-Cloud/Deployment/Hotfixes/Move-files-manually/index.md

```diff
@@ -37,44 +37,46 @@ For the sake of simplicity here's an explanation of the names I'll be using in t
 
 ### Move the files
 
-1. Clone down the Live environment
+1. Clone down the Live environment.
     * The _clone URL_ for the Live environment can be found in the Umbraco Cloud Portal:
 
-        ![Live Clone URL](images/live-clone-url.png)
+        ![Live Clone URL](images/live-clone-url_v10.png)
 
-2. Locate the files from the Development repository that you want to move to Live
-    * Check the commits in the Git history for the Development repository to verify which files you need
+2. Locate the files from the Development repository that you want to move to Live.
+    * Check the commits in the Git history for the Development repository to verify which files you need.
 
         ![Files changes or added](images/commit-files-changed.png)
 
-    * The _new files_ can be moved from the Development repository to the Live repository
-    * The same goes for _changed files_. You can also edit the files, and only move the code snippets you need on the Live environment
+    * The _new files_ can be moved from the Development repository to the Live repository.
+    * The same goes for _changed files_. You can also edit the files, and only move the code snippets you need on the Live environment.
 
-3. Copy and paste the new and/or updated files from your Development repository to your Live repository
-4. You can now _Stage_ and _Commit_ these changes to the Live repository in Git
+3. Copy and paste the new and/or updated files from your Development repository to your Live repository.
+4. You can now _Stage_ and _Commit_ these changes to the Live repository in Git.
 
-One of the benefits of having the Live environment cloned down, is that you can test the new changes locally before sending it to the Live environment.
+One of the benefits of having the Live environment cloned down, is that you can test the new changes locally before sending them to the Live environment.
 
 ### Test changes locally
 
-5. Run the Live repository through IIS
-6. Open _CMD_ and navigate to the `/data/revision` folder in your Live repository
-7. Create a _deploy_ marker by typing the following: `echo > deploy` - learn more about this command in the [Power Tools](../../../Set-up/Power-Tools) articles
-8. The changes will now be reflected in the backoffice of your local Live environment
+1. Run the Live repository through IIS
+2. Open _CMD_ and navigate to the `/umbraco/Deploy/Revision` folder in your Live repository.
```

Comment:
I am guessing since this article is not only for v9+ we should still say where you can find the revision folder in v8 as well, otherwise people might get confused if they can't find the revision folder in the Umbraco folder :)

Reply:
Good point.. I'll add it.